packages/driver/cypress/integration/e2e/multi-domain/navigation_spec.ts

```diff
@@ -199,6 +199,27 @@ describe('navigation events', { experimentalSessionSupport: true }, () => {
   })
 })
 
+// @ts-ignore / session support is needed for visiting about:blank between tests
+describe('event timing', { experimentalSessionSupport: true }, () => {
+  it('does not timeout when receiving a delaying:html event after switchToDomain has started, but before the spec bridge is ready', () => {
+    cy.visit('/fixtures/multi-domain.html')
+    cy.get('a[data-cy="multi-domain-secondary-link"]').click()
+
+    cy.switchToDomain('http://foobar.com:3500', () => {
+      cy.log('inside switchToDomain foobar')
+    })
+
+    // Honestly it's surprising this command runs, this command is run from localhost but executes against a cross origin aut (foobar.com)
```

Comment:
The `href` property is writable from a cross-origin frame.

https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy#location

Reply:
Suggest removing this comment or updating to reference the mdn docs.

Reply:
updated